cocotb/share/makefiles/simulators/Makefile.verilator

```diff
@@ -0,0 +1,110 @@
+###############################################################################
+# Copyright (c) 2013 Potential Ventures Ltd
+# Copyright (c) 2013 SolarFlare Communications Inc
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#     * Redistributions of source code must retain the above copyright
+#       notice, this list of conditions and the following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in the
+#       documentation and/or other materials provided with the distribution.
+#     * Neither the name of Potential Ventures Ltd,
+#       SolarFlare Communications Inc nor the
+#       names of its contributors may be used to endorse or promote products
+#       derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL POTENTIAL VENTURES LTD BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+###############################################################################
+
+ifneq ($(VHDL_SOURCES),)
+
+results.xml:
+	@echo "Skipping simulation as VHDL is not supported on simulator=$(SIM)"
+debug: results.xml
+clean::
+
+else
+
+CMD := verilator
+
+ifeq ($(shell which $(CMD) 2>/dev/null),)
+# Verilator is not in PATH, lets start searching for it
```

Comment:
We don't do such searching for any other simulator; I'd be inclined to not do it here either to keep things simple and easy to document.

Do you have a strong use case for it?

Reply:
Those are the three supported ways of installing and using Verilator and I believe we should support them too.

Reply:
Can't we just leave the user responsible for putting `verilator` on the path, which in my limited experience is how runtime dependencies always work?

Reply:
We could, but what would it solve? There are those three ways of finding Verilator defined by the tool itself, searching for verilator in the path is the highest precedence. If not, it will search the other ones. I am generally okay with removing it, but I cannot see how it improves quality..

Reply:
We could, but what would it solve? There are those three ways of finding Verilator defined by the tool itself, searching for `verilator` in the path is the highest precedence. If not, it will search the other ones. I am generally okay with removing it, but I cannot see how it improves quality..

Reply:
We could, but what would it solve? There are those three ways of finding Verilator defined by the tool itself, searching for `verilator` in the path is the highest precedence. If not, it will search the other ones. I am generally okay with removing it, but I cannot see how it improves quality..

Reply:
Removed and amended